trie/triedb/hashdb/database.go

```diff
@@ -617,7 +591,7 @@ func (db *Database) Update(root common.Hash, parent common.Hash, nodes *trienode
 			if err := rlp.DecodeBytes(n.Blob, &account); err != nil {
 				return err
 			}
-			if account.Root != db.EmptyRoot() {
+			if account.Root != types.EmptyRootHash {
```

Comment:
Doesn't it need to be imported through `types.GetEmptyRootHash(bool)`?

Reply:
(https://github.com/kroma-network/go-ethereum/commit/b3bf8f33ccd28b0a4e1110090afa194bf1c6e201) After this commit, `hashdb.Database` will only use mpt, so there is no need to call `db.EmptyRoot()`